modules/utxo-ord/test/inscription.ts

```diff
@@ -34,4 +34,29 @@ describe('inscriptions', () => {
       );
     });
   });
+
+  describe('Inscription Reveal Data', () => {
+    it('should sign reveal transaction and validate reveal size', () => {
+      const ecPair = ECPair.makeRandom();
+      const inscriptionData = Buffer.from('And Desert You', 'ascii');
+      const { revealTransactionVSize, tapLeafScript, address } = inscriptions.createInscriptionRevealData(
+        ecPair.publicKey,
+        contentType,
+        inscriptionData,
+        networks.testnet
+      );
+
+      const randomHash = '96b2376fb0ccfdbcc9472489ca3ec75df1487b08a0ea8d9d82c55da19d8cceea';
+      const fullySignedRevealTransaction = inscriptions.signRevealTransaction(
+        ecPair.privateKey as Buffer,
+        tapLeafScript,
+        randomHash,
```

Comment:
Don't we need to specify the `vout` / `index` here in addition to the `hash` for the input from the commit transaction?